docs/source/deployment.rst

```diff
@@ -54,6 +60,29 @@ To speed up the Terraform run, the module name may be specified with the ``targe
   $ python manage.py build --target kinesis       # tf_stream_alert_kinesis module
   $ python manage.py build --target stream_alert  # tf_stream_alert module
 
+To apply specific terraform changes, use ``list-target`` and ``build`` commands following with ``--target`` option:
```

Comment:
I'm so sorry to make you update this again - but typo here: `list-target` should be `list-targets`